ArchiSteamFarm/IPC/ArchiKestrel.cs

```diff
@@ -122,7 +122,7 @@ internal static class ArchiKestrel {
 					webBuilder.UseKestrel(static (builderContext, options) => options.Configure(builderContext.Configuration.GetSection("Kestrel")));
 				} else {
 					// Use ASF defaults for Kestrel
-					webBuilder.UseKestrel(static options => options.ListenLocalhost(1242));
```

Comment:
No.

Reply:
I love it when debug code makes it into the commit... not